lua/possession/session.lua

```diff
@@ -374,14 +379,17 @@ function M.list(opts)
     local files_by_name = {}
 
     local sessions = {}
+    local cwd = vim.fn.getcwd(-1, -1)
```

Comment:
(besides the above comment about filtering here)

This gives us the global CWD, but I think it would be better to use local CWD. If user didn't `:tcd` or `:lcd` then this is the same as global CWD, but if they did, then I think we should respect that when filtering by CWD.